### PR TITLE
Updating README.md.

### PR DESCRIPTION
**Pull request by dsuhinin**

### CODE REVIEW

File: README_NEW.md [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+# Add the UI back to it
+tar -C .mlflow.repo/mlflow -xzvf ./ui.tgz
+
+# Install it in editable mode
+pip install -e .mlflow.repo
+```
+
+or run:
+```bash
+mage configure
```
Comment (suprjinx) [inline README_NEW.md:82]: seems like this should be the first, not alternate, option, since so much simpler

File: README_NEW.md [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+
+Once the mapping has been done, validation will be invoked automatically in the generated fiber code.
+
+When the need arises, we can write custom validation function in [pkg/validation/validation.go](./validation/validation.go).
+
+### Data access
+
+Initially, we want to focus on supporting Postgres SQL. We chose [Gorm](https://gorm.io/) as ORM to interact with the database.
+
+We do not generate any Go code based on the database schema. Gorm has generation capabilities but they didn't fit our needs. The plan would be to eventually assert the current code stil matches the database schema via an intergration test.
```
Comment (suprjinx) [inline README_NEW.md:199]: typo: stil

File: README_NEW.md [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+mage
+
+# Execute single target
+mage generate
+```
+
+The beauty of Mage is that we can use regular Go code for our scripting.  
+That being said, we are not married to this tool.
+
+## Development Setup
```
Comment (suprjinx) [inline README_NEW.md:56]: Things get pretty complicated at this point -- maybe move to a separate `DEVELOPER.md`?

File: README_NEW.md [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+  * [Client-side Go implementation](#client-side-go-implementation)
+  * [Go store in Python](#go-store-in-python)
+* [Misc](#misc)
+  * [Debug Failing Tests](#debug-failing-tests)
+  * [Supported endpoints](#supported-endpoints)
+  * [Targeting Local Postgres in Python Tests](#targeting-local-postgres-in-python-tests)
+
+## Installation
+
+This package is not yet available on PyPI and currently requires the [Go SDK](https://go.dev) to be installed.
```
Comment (suprjinx) [inline README_NEW.md:31]: Why don't we push builds to PyPI so this is simpler?

Comment (nojaf) [reply README_NEW.md:31]: We first need to move to https://github.com/mlflow/mlflow-go and set it up over there.
Might require helps from Databricks.

File: README_NEW.md (hunk 3 above)
Comment (nojaf) [reply README_NEW.md:56]: Yes, this probably should be in a `CONTRIBUTING.md` file.

File: README_NEW.md [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+```
+
+The beauty of Mage is that we can use regular Go code for our scripting.  
+That being said, we are not married to this tool.
+
+## Development Setup
+
+### Installation
+
+```bash
```
Comment (nojaf) [inline README_NEW.md:60]: Consider rearranging the content to place `mage configure` at the top. Start with a brief explanation of its purpose, and then use an expandable section to provide more detailed information about its functionality.

File: README_NEW.md [hunk 6]
```diff
@@ -0,0 +1,349 @@
...
+# If you want to run a specific test with more verbosity
+# -s for live output
+# --log-level=debug for more verbosity (passed down to the Go server/stores)
+MLFLOW_GO_LIBRARY_PATH=$libpath pytest --confcutdir=. \
+  .mlflow.repo/tests/tracking/test_rest_tracking.py::test_create_experiment_validation \
+  -k 'not [file' \
+  -s --log-level=debug
+```
+
+next `mage` tests targets are available:
```
Comment (nojaf) [inline README_NEW.md:132]: Same, mage targets first, with expandable sections detailing their actions.

File: README_NEW.md [hunk 7]
```diff
@@ -0,0 +1,349 @@
...
+
+```bash
+mage generate
+```
+
+Go code will be generated. Use the protos files from `.mlflow.repo` repository.
+
+This includes the generation of:
+
+- Structs for each endpoint. ([pkg/protos](./protos/service.pb.go))
```
Comment (nojaf) [inline README_NEW.md:180]: These links no longer work.

File: README_NEW.md [hunk 8]
```diff
@@ -0,0 +1,349 @@
...
+
+```bash
+pre-commit run golangci-lint --all-files
+```
+
+Sometimes `golangci-lint` can complain about unrelated files, run `golangci-lint cache clean` to clear the cache.
+
+## Usage
+
+### Go server
```
Comment (nojaf) [inline README_NEW.md:215]: For developers it would be interesting to mention `mage dev`, that is how I always start the thing.

**Pull request closed without merging**